checklists/alz_checklist.en.json

```diff
@@ -555,12 +544,56 @@
             "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/azure-best-practices/track-costs"
         },
+        {
+            "category": "Resource Organization",
+            "subcategory": "Regions",
+            "text": "Select the right Azure region/s for your deployment. Azure is a global-scale cloud platform that provide global coverage through many regions and geographies. Different Azure regions have different characteristics, access and availability models, costs, capacity, and services offered, then it is important to consider all criteria and requirements",
+            "waf": "Reliability",
+            "guid": "250d81ce-8bbe-4f85-9051-6a18a8221e50",
+            "id": "C03.01",
+            "severity": "High",
+            "training": "https://learn.microsoft.com/learn/modules/azure-architecture-fundamentals/",
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/azure-setup-guide/regions"
+        },
+        {
+            "category": "Resource Organization",
+            "subcategory": "Regions",
+            "text": "Consider a multi-region deployment. Depending on customer size, locations, and users presence, operating in multiple regions can be a common choice to deliver services and run applications closer to them. Using a multi-region deployment is also important to provide geo disaster recovery capabilities, to eliminate the dependency from a single region capacity and diminish the risk of a temporary and localized resource capacity constraint",
+            "waf": "Reliability",
+            "guid": "19ca3f89-397d-44b1-b5b6-5e18661372ac",
+            "id": "C03.02",
+            "severity": "Medium",
+            "training": "https://learn.microsoft.com/learn/modules/azure-architecture-fundamentals/",
+            "link": "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-setup-guide/regions#operate-in-multiple-geographic-regions"
+        },
+        {
+            "category": "Resource Organization",
+            "subcategory": "Regions",
+            "text": "Ensure required services and features are available within the chosen deployment regions",
+            "waf": "Reliability",
+            "guid": "4c27d42e-8bba-4c75-9155-9ab9153e8908",
+            "id": "C03.03",
+            "severity": "Medium",
+            "training": "https://learn.microsoft.com/learn/modules/azure-architecture-fundamentals/",
+            "link": "https://azure.microsoft.com/explore/global-infrastructure/products-by-region/"
+        },
+        {
+            "category": "Resource Organization",
+            "subcategory": "Regions",
+            "text": "Ensure resource quotas for every resource type, per subscription and per region, have been requested and approved time before a new deployment or migration. Back-office process can take time to be approved and could delay deployment execution",
```

Comment:
Hi @igorpag, we have a checklist item to ensure resource quotas are monitored / customer is aware of them. I feel this item is covered. 
Think it makes sense to remove this one to avoid confusion and duplication.